modules/clientBinaryManager.js

```diff
@@ -180,7 +180,9 @@ class Manager extends EventEmitter {
         .then((localConfig) => {
             if (!localConfig) {
                 log.info('No config for the ClientBinaryManager could be loaded, using local clientBinaries.json.');
-                localConfig = require('../clientBinaries.json');
+
+                const localConfigPath = path.join(Settings.userDataPath, 'clientBinaries.json');
```

Comment:
I dont understand what is the fix. On line 87 it already tries to load the local config. Then later it will replace it with the remote loaded one, if one came if not it keeps the locally loaded one. On this line it basically already checked local and remote ones and therefore loads the one from the mist folder.